modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/context/index.js

```diff
@@ -13,11 +15,13 @@ import reducer, {actionTypes} from './reducer';
 const AppContext = createContext();
 
 const AppContextProvider = ({assetsPath, children, page}) => {
+	const {oktaSessionURL} = useApplicationProvider();
 	const dispatchEvent = useCustomEvent(CUSTOM_EVENTS.USER_ACCOUNT);
 	const [state, dispatch] = useReducer(reducer, {
 		assetsPath,
 		page,
 		project: {},
+		sessionId: 'sessionId',
```

Comment:
```suggestion
		sessionId: '',
```

Reply:
If we remove the mock, the pages won't work until the Okta is finished, so I thought that using that for a while.